raiden_installer/tokens.py

```diff
@@ -68,7 +68,11 @@ def address(self) -> str:
     def find_by_ticker(ticker):
         major, minor, _ = CONTRACTS_VERSION.split(".", 2)
         version_string = f"{major}.{minor}"
-        token_list_version = {"0.25": TokensV25, "0.33": TokensV33}.get(version_string, Tokens)
+        token_list_version = {"0.25": TokensV25,
```

Comment:
why do we need so many tokens?

Reply:
Right now you're able to specify any raiden version that you like. This means that also any Token version should be supported. I already added an issue about reformatting the whole Token handling in Issue #104

Reply:
We should only support 1 Raiden Version on Mainnet and one for the Demo-Env